mimic/test/test_maas.py

```diff
@@ -145,11 +145,27 @@ def test_list_entity(self):
         """
         test list entity
         """
+        marker = ""
         req = request(self, self.root, "GET", self.uri + '/entities', '')
         resp = self.successResultOf(req)
         self.assertEquals(resp.code, 200)
         data = self.get_responsebody(resp)
         self.assertEquals(data['metadata']['count'], 1)
+        for q in range(1, 101):
+            self.createEntity('Cinnamon' + str(q))
+        for q in range(1, 101):
+            req = request(self, self.root, "GET", self.uri + '/entities/?limit=' + str(q), '')
```

Comment:
nit: dont have to pass the `' '` to indicate that there is no body for the `GET`